lib/connection/connection_config.js

```diff
@@ -81,7 +81,9 @@ function consolidateHostAndAccount(options) {
     if (options.region === 'us-west-2') {
       options.region = '';
     }
-    if (dotPos < 0 && Util.isString(options.region) && options.region.length > 0) {
+    if (Util.exists(options.host)) {
```

Comment:
Could you please add some test cases for that (file `test/unit/connection/connection_config_test.js`).

Reply:
i am unable to run tests